### PR TITLE
feat: add variable for aliyun theme of modal

### PR DESCRIPTION
**Pull request by hi-zhaoyeqing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (add variable for aliyun theme of modal)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |feat: add variable for aliyun theme of modal|
| 🇨🇳 Chinese |新功能：为modal组件添加阿里云主题变量|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/modal/style/confirm.less [hunk 1]
```diff
@@ -23,7 +23,7 @@
       overflow: hidden;
       color: @heading-color;
       font-weight: 500;
-      font-size: @font-size-lg;
+      font-size: @modal-confirm-body-title-font-size;
```
Comment (vthinkxie) [inline components/modal/style/confirm.less:26]: modal-confirm-title-font-size 是不是更好一些

File: components/modal/style/confirm.less [hunk 2]
```diff
@@ -40,7 +40,7 @@
 
       // `content` after `icon` should set marginLeft
       + .@{confirm-prefix-cls}-title + .@{confirm-prefix-cls}-content {
-        margin-left: 38px;
+        margin-left: @modal-confirm-body-content-icon-margin-left;
```
Comment (vthinkxie) [inline components/modal/style/confirm.less:43]: modal-confirm-icon-margin-left ？

File: components/style/themes/variable.less [hunk 3]
```diff
@@ -618,6 +618,9 @@ html {
 @modal-footer-border-width: @border-width-base;
 @modal-mask-bg: fade(@black, 45%);
 @modal-confirm-body-padding: 32px 32px 24px;
+@modal-confirm-icon-margin-left: 38px;
+@modal-confirm-title-font-size: @font-size-lg;
+
```
Comment (vthinkxie) [inline components/style/themes/variable.less:623]: 多了一行回车

**Review by vthinkxie — APPROVED** [review]
lgtm

File: components/modal/style/confirm.less [hunk 4]
```diff
@@ -40,7 +40,7 @@
 
       // `content` after `icon` should set marginLeft
       + .@{confirm-prefix-cls}-title + .@{confirm-prefix-cls}-content {
-        margin-left: 38px;
+        margin-left: @modal-confirm-icon-margin-left;
```
Comment (afc163) [inline components/modal/style/confirm.less:43]: 这个现在是写死的，但其实是应该从』图标宽度 + 图标右边距』算出来。

Comment (qdzhaoxiaodao) [reply components/modal/style/confirm.less]: 已修改

File: components/style/themes/variable.less (hunk 3 above)
Comment (qdzhaoxiaodao) [reply components/style/themes/variable.less:622]: 已修改

File: components/modal/style/confirm.less (hunk 2 above)
Comment (qdzhaoxiaodao) [reply components/modal/style/confirm.less]: 已修改

File: components/modal/style/confirm.less (hunk 1 above)
Comment (qdzhaoxiaodao) [reply components/modal/style/confirm.less]: 已修改

**Review by afc163 — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

**Pull request merged**